<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Board</class>
 <widget class="QWidget" name="Board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #cfa844</string>
  </property>
  <widget class="QPushButton" name="btnUndo">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>25</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Battle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSettings">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>65</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="choosePanel" native="true">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnBlack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWhite">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTwoTurns">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Two more turns</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_off">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>71</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Offensive
Player</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_def">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>71</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Player
Defensive</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_off_name">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Human</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_def_name">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>NaiveAI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_off_score">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_def_score">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>Board</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
